getting_started/step_by_step/singletons_autoload.rst

```diff
@@ -151,6 +181,40 @@ current scene and replaces it with the requested one.
         # optional, to make it compatible with the SceneTree.change_scene() API
         get_tree().set_current_scene( current_scene )
 
+ .. code-tab:: csharp
+
+    public void GotoScene(string path)
+    {
+        // This function will usually be called from a signal callback,
+        // or some other function from the running scene.
+        // Deleting the current scene at this point might be
+        // a bad idea, because it may be inside of a callback or function of it.
+        // The worst case will be a crash or unexpected behavior.
+
+        // The way around this is deferring the load to a later time, when
+        // it is ensured that no code from the current scene is running:
+
+        CallDeferred("DeferredGotoScene", path);
```

Comment:
It's not mandatory, but probably we could consider using `nameof(DeferredGotoScene)` here, since it's a good practice which is still supported in C# 6.